calc-dupe-candidates.sh

```diff
@@ -0,0 +1,3 @@
+cd /home/ubuntu/mergereduce/src/compare;
+export PYSPARK_PYTHON=/usr/bin/python3;
+spark-submit --master spark://ip-10-0-0-7:7077 --conf spark.executor.memoryOverhead=3000  --executor-memory 6G --driver-memory 6G compare.py
```

Comment:
IPs shouldn't be in the repo

Reply:
@coruscating internal ones too? Should I put it in a config file or is there an easier way to do this?

Reply:
@coruscating I removed it and made it a command line argument